velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -417,6 +417,38 @@ TEST_F(ArithmeticTest, isNanDouble) {
   EXPECT_EQ(false, isNan(std::nullopt));
 }
 
+TEST_F(ArithmeticTest, hexWithBigint) {
+  const auto toHex = [&](std::optional<int64_t> value) {
+    return evaluateOnce<std::string>("hex(c0)", value);
+  };
+  EXPECT_EQ(std::nullopt, toHex(std::nullopt));
+  EXPECT_EQ("11", toHex(17));
+  EXPECT_EQ("0", toHex(0));
+  EXPECT_EQ("FFFFFFFFFFFFFFFF", toHex(-1));
```

Comment:
Would you also add toHex(-17) to illustrate how it differs from toHex(17)?

Reply:
already added.

Reply:
Somehow, I don't see an example with -17